client/menu.cpp

```diff
@@ -258,7 +258,7 @@ void gov_menu::update_all()
 /**
    Instantiate a new goto and act sub menu.
  */
-go_act_menu::go_act_menu(QWidget *parent) : QMenu(_("Go to and..."), parent)
+go_act_menu::go_act_menu(QWidget *parent) : QMenu(_("Goto And..."), parent)
```

Comment:
Should we really merge `goto` or is it informal usage? My spellchecker isn't happy.

Reply:
I did a quick search, seems "go to" is the correct way. I'll fix.